### PR TITLE
DSR-166: remove need for contentful.json file

### PR DESCRIPTION
**Pull request by rupl**
## https://humanitarian.atlassian.net/browse/DSR-166

For @teodorescuserban to merge only. I believe it works without changing the stack but he needs to confirm that. Marking as draft PR until confirmed.

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
   env: {
-    CTF_HOST: api.CTF_HOST || 'cdn.contentful.com',
-    CTF_SPACE_ID: api.CTF_SPACE_ID,
-    CTF_ENVIRONMENT: api.CTF_ENVIRONMENT || 'master',
-    CTF_CDA_ACCESS_TOKEN: api.CTF_CDA_ACCESS_TOKEN,
-    CTF_CDA_PREVIEW_TOKEN: api.CTF_CDA_PREVIEW_TOKEN,
-    baseUrl: api.BASE_URL || 'https://reports.unocha.org',
+    CTF_HOST: process.env.CTF_HOST || 'cdn.contentful.com',
+    CTF_SPACE_ID: process.env.CTF_SPACE_ID,
+    CTF_ENVIRONMENT: process.env.CTF_ENVIRONMENT || 'master',
```
Comment (teodorescuserban) [inline nuxt.config.js]: Some of the defaults (although fake) are already provided in the Dockerfile. The `CTF_ENVIRONMENT` for example.
You decide where do you want to put them but lets group them up in either one.

File: nuxt.config.js [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
+    CTF_CDA_ACCESS_TOKEN: process.env.CTF_CDA_ACCESS_TOKEN,
+    CTF_CDA_PREVIEW_TOKEN: process.env.CTF_CDA_PREVIEW_TOKEN,
+    baseUrl: process.env.BASE_URL || 'https://reports.unocha.org',
```
Comment (teodorescuserban) [inline nuxt.config.js]: I'd leave that as the local dev url rather than prod.

File: nuxt.config.js (hunk 1 above)
Comment (rupl) [reply nuxt.config.js]: Keeping them in docker makes sense to me. I'll throw one more commit up here to remove all the JS "defaults"

File: Dockerfile [hunk 3]
```diff
@@ -8,11 +8,6 @@ WORKDIR /srv/www
 COPY . .
 
 RUN apk add -U \
```
Comment (teodorescuserban) [inline Dockerfile]: @rupl remove this as well (keep `RUN`) or else it will throw errors.

should start with 
`RUN yarn install && \`

**Pull request merged**